src/pages/Home.js

```diff
@@ -181,124 +187,177 @@ function Home() {
   };
   // Pagination
 
-  let Foodspots = () => {
-    return currentRecords.map((item, index) => {
-      return (
-        <Link to={`/foodspot/` + item.id} state={item.id} key={item.id}>
-          <li key={index}>{item.title}</li>
-        </Link>
-      );
-    });
-  };
-
   return (
-    <div className="flex justify-center my-[30px]">
-      <div className="filter-section">
-        <div className="filter-buttons">
+    <div className="contentWrapper max-w-[1915px] mx-auto my-[100px] flex px-[100px]">
```

Comment:
müssen wir hier mit  'max-w-[1915px] my-[100px] flex px-[100px]'  arbeiten? ich würde eher auf globale standard Definitionen zurückgreifen. Also schauen ob wir mit den Standardwerten auskommen oder eben in der tailwind config neu hinzufügen, die dann auch an anderen Stellen wiederverwendet werden können.  In größeren Projekten führt das sonst zu Inkonsistenzen, weil überall andere Werte verwendet werden. Außerdem wird dadurch der CSS Output vergrößert. Das just in time feature von Tailwind würde ich daher nur in Spezialfeällen benutzen. Also in wirklichen Einzelfällen und ansonsten versuchen mit den Standards auszukommen oder diese zu erweitern.